rustler_mix/lib/rustler.ex

```diff
@@ -75,7 +75,7 @@ defmodule Rustler do
           load_data: :something
       end
 
-  [target]: https://forge.rust-lang.org/release/platform-support.html
+  [target]: https://doc.rust-lang.org/nightly/rustc/platform-support.html
```

Comment:
Why nightly?

Reply:
It was the first link redirected from that old one. I changed to be `stable`.